department-of-veterans-affairs/vets-website
# Pension mvp review and submit

## Summary
Update the 'Review application' page in the Pension Benefits application(21P-527EZ) to validate and edit entire application

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Sign in via mocked authentication at `http://localhost:3001/sign-in/mocked-auth`
3. Start or continue the application
4. Go to the 'Review application' page (/review-and-submit)
5. Verify that all chapters and pages can be edited before submitting the form
6. Verify that all chapters and pages are validated before submitting the form

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71857)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots
| Review and submit | Expanded | Invalid |
| -------- | -------- | -------- |
|![localhost_3001_pension_application_527EZ_review-and-submit](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/35141ece-9f61-4487-97ed-fb3bad7a1511)|![localhost_3001_pension_application_527EZ_review-and-submit (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/08336288-f1ef-4bfb-838a-01a711cb2324)|![localhost_3001_pension_application_527EZ_review-and-submit (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/139bf832-9fa7-4b1c-9b11-03e5870e2623)|


## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71857)

### Quality Assurance & Testing

### Error Handling


### Authentication
